### ISSUE TITLE
Accessibility: improve by hooking into phone settings

### ISSUE THREAD

**Issue by brandyscarney** [ionic-team/ionic#8780]
We should check the user's settings for accessibility and modify the framework components based on it: https://github.com/driftyco/ionic/issues/8743

For example, if button shapes are enabled the clear button should change to the following:

![slack for ios upload-5](https://cloud.githubusercontent.com/assets/6577830/19488926/0c1fdba8-9537-11e6-9f78-2f343083c83b.jpg)

**Comment by jgw96** [comment]
This plugin could possibly help with this https://github.com/phonegap/phonegap-mobile-accessibility
